morphir-decoration-extension/src/getNonce.ts

```diff
@@ -0,0 +1,9 @@
+export function getNonce() {
```

Comment:
What's the purpose of this? Can you add some comments?

Reply:
I forgot to clean that up, It is currently not being used